### PR TITLE
Sync

### PR DESCRIPTION
**Pull request by ronualdo**
Changes to make tutorial a little bit more similar to the reference

### CODE REVIEW

File: tutorial/index.jade [hunk 1]
```diff
@@ -1,10 +1,155 @@
...
+
+      1. The user sees a success page and receives an SMS indicating that their account has been created
+
+      ## Building Blocks
+
+      To get this done, you'll be working with the following Twilio-powered APIs:
+
+      **Authy REST API**
+
+      * [Users Resource](https://docs.authy.com/#section-Enabling_two-factor_on_a_user): You will need to create Authy users to send and verify one-time passwords with them.
```
Comment (acamino) [inline tutorial/index.jade]: Nitpick: You should use **protocol relative URL**. Would you mind updating all the URLs in this tutorial to accordingly?

File: tutorial/index.jade [hunk 2]
```diff
@@ -1,10 +1,155 @@
...
+    :markdown
+      ## Introduction
+
+      Ready to implement user account verification in your application? Here's how it works at a high level:
+
+      1. The user begins the registration process by entering their data, including a phone number, into a signup form
+
+      1. The authentication system sends a one-time password to the user's mobile phone to verify their possession of that phone number
+
+      1. The user enters the one-time password into a form before completing registeration
```
Comment (acamino) [inline tutorial/index.jade]: Fix the typo in _registeration_ please?

File: tutorial/index.jade [hunk 3]
```diff
@@ -1,10 +1,155 @@
...
+
+  .step(data-title='The Form Template', data-file='views/users/create.jade', data-language='jade')
+    :markdown
+      ## The Form Template
+
+      When we create a new user, we ask for a name, e-mail address, and a password. 
+      In order to validate their account we also ask them for a mobile number with a country code. 
+      We use Authy to send them a one-time password via SMS to this phone number.
+     
+      Now the user is logged in but not verified. In the next steps we'll learn how to verify the user using authy.
```
Comment (acamino) [inline tutorial/index.jade]: _authy_ is a proper noun, it should be capitalised. Could you please do this change?

File: tutorial/index.jade [hunk 4]
```diff
@@ -1,10 +1,155 @@
...
+
+      We've just implemented account verification so your users can
+      confirm their phone number. If you're a Node developer working with Twilio,
+      you might want to check out these other tutorials:
+
+      [**Click-To-Call**](//www.twilio.com/docs/tutorials/walkthrough/click-to-call/node/express)
+
+      Put a button on your web page that connects visitors to live support or sales people via telephone.
+
+      [**Automated Survey**](//https://www.twilio.com/docs/tutorials/walkthrough/automated-survey/node/express)
```
Comment (acamino) [inline tutorial/index.jade]: Looks like this one is broken. 😯

File: tutorial/index.jade [hunk 5]
```diff
@@ -1,10 +1,155 @@
...
+
+      1. The user begins the registration process by entering their data, including a phone number, into a signup form
+
+      1. The authentication system sends a one-time password to the user's mobile phone to verify their possession of that phone number
+
+      1. The user enters the one-time password into a form before completing registeration
+
+      1. The user sees a success page and receives an SMS indicating that their account has been created
+
+      ## Building Blocks
```
Comment (acamino) [inline tutorial/index.jade:18]: Could _Building Blocks_ be a new section? Seems like the **Introduction** section is quite big.

File: tutorial/index.jade (hunk 3 above)
Comment (acamino) [inline tutorial/index.jade]: Usually is a good practice to add references to increase the visibility to the tutorial. That's why we have the _see also_ section in most of the steps. Would you mind adding a reference to [Jade](http://jade-lang.com) please?

File: tutorial/index.jade [hunk 6]
```diff
@@ -1,10 +1,155 @@
...
+    :markdown
+      ## Verifying the Code: Controller
+
+      This controller function handles the form submission. It's a little longer than the others, but it has a lot to do. It needs to:
+
+      * Load a `User` model for the current verification request
+      * Use an instance function on the model object to verify the code that was entered by the user
+      * If the code entered was valid, flip a boolean flag on the user model to indicate the account was verified
+
+      Let's look at the `User` model to check out the instance method that handles verifying the code with Authy next.
```
Comment (acamino) [inline tutorial/index.jade:95]: @ronualdo what about adding some reference to the [`Authy#verify`](https://www.npmjs.com/package/authy#verify-token) method. This one is used under the hood at `verifyAuthyToken`.

Also adding a reference to [`Model#save`](http://mongoosejs.com/docs/api.html#model_Model-save) would be good. 

I'd appreciate hearing your thoughts on this.

File: tutorial/index.jade [hunk 7]
```diff
@@ -1,10 +1,155 @@
...
+      We call Authy, and then immediately call a passed callback function with the result.
+
+  .step(data-title='Re-sending The Code', data-file='controllers/users.js', data-highlight='65-93')
+    :markdown
+      ## Re-sending the Code
+
+      This controller function loads the `User` model associated with the request and then uses the same instance function we defined earlier to resend the code. Pretty straightforward!
+
+      To wrap things up, let's implement the last step where we confirm that the user's account has
+      been verified with a text message and a success page.
```
Comment (acamino) [inline tutorial/index.jade:117]: This section uses a flash messages middleware. Would you mind adding a reference for it?

Please take a look at https://github.com/jaredhanson/connect-flash#connect-flash.

File: tutorial/index.jade [hunk 8]
```diff
@@ -1,10 +1,155 @@
...
+
+  .step(data-title="Show User Details Page", data-file="controllers/users.js", data-highlight="149-167")
+    :markdown
+      ## Show User Details Page
+
+      This controller function renders a Jade template that contains the user's full name, and indicates whether or not they are verified by checking the user's `verified` property.
+
+      This should suffice for confirmation in the browser that the user has been verified. Let's see how we might send that text message next.
```
Comment (acamino) [inline tutorial/index.jade:128]: Adding a reference to [`Model#findById`](http://mongoosejs.com/docs/api.html#model_Model.findById) would be good here.

**Comment by acamino** [comment]
Thanks for the changes @ronualdo. Could you please squash the commits?

**Comment by ronualdo** [comment]
I don't really see a problem with the commits. IMHO all of them are isolated and descriptive about the changes that they are adding. Still not sure if we really need to squash that.

**Comment by acamino** [comment]
I think you're made your point. Thanks @ronualdo!

**Pull request merged**